src/main.rs

```diff
@@ -188,8 +319,34 @@ unsafe extern "C" fn print(cx: *mut JSContext, argc: u32, vp: *mut Value) -> boo
     return true;
 }
 
+unsafe extern "C" fn load(cx: *mut JSContext, argc: u32, vp: *mut Value) -> bool {
+    let args = CallArgs::from_vp(vp, argc);
+    for i in 0..args._base.argc_ {
+        let val = args.get(i);
+        let s = js::rust::ToString(cx, val);
+        if s.is_null() {
+            // report error
+            return false;
+        }
+        let mut filename = env::current_dir().unwrap();
+        let path_root = Rooted::new(cx, s);
+        let path = JS_EncodeStringToUTF8(cx, path_root.handle());
+        let path = CStr::from_ptr(path);
+        filename.push(str::from_utf8(path.to_bytes()).unwrap());
+        let global = CurrentGlobalOrNull(cx);
+        let global_root = Rooted::new(cx, global);
+        RUNTIME.with(|ref r| {
+            let _ = run_script(r.borrow().as_ref().unwrap(), global_root.handle(),
```

Comment:
We really should propagate errors from under `run_script`, and for this case abort when an error occurs. Doesn't need to block landing this, though.